service/history/api/respondworkflowtaskcompleted/api.go

```diff
@@ -129,6 +130,12 @@ func (handler *WorkflowTaskCompletedHandler) Invoke(
 				metrics.StaleMutableStateCounter.With(handler.metricsHandler).Record(
 					1,
 					metrics.OperationTag(metrics.HistoryRespondWorkflowTaskCompletedScope))
+				softassert.Sometimes(handler.logger).Debug("stale mutable state detected",
+					tag.WorkflowID(token.GetWorkflowId()),
```

Comment:
I noticed in logs I added that when I added the `tag.WorkflowID` it ended up duplicated, do you know if we have a way to de-dupe those?

Reply:
More of an annoyance than anything, certainly not blocking this PR